src/lib/braze.ts

```diff
@@ -0,0 +1,23 @@
+import { data as sd } from "sharify"
+
+export const setup = async () => {
+  if (
+    typeof window === "undefined" ||
+    !sd.BRAZE_API_KEY ||
+    !sd.BRAZE_API_URL ||
+    process.env.NODE_ENV !== "production"
+  ) {
+    return
+  }
```

Comment:
Totally non-blocking! but reading all of the conditions in the if was a little confusing to me. Maybe something like the below, where we declare the _good_ conditions and then negate:

```js
const shouldInit = isClient && getENV('BRAZE_API_KEY') && getENV('BRAZE_API_URL')
if (!shouldInit) {
  return 
}
```

Also, maybe keep it on in dev? I could see it being confusing to debug later if its guarded by a prod flag. Or better yet, maybe there's a `.env` variable we can set to toggle.